src/js/components/SkipLinks/SkipLinks.js

```diff
@@ -35,29 +37,38 @@ const SkipLinks = ({ children, id, messages }) => {
   return (
     <Layer
       id={id}
-      position={showLayer ? 'top' : 'hidden'}
+      position={showLayer ? theme.skipLinks.position : 'hidden'}
       ref={layerRef}
       onFocus={onFocus}
       onBlur={onBlur}
+      tabIndex={0}
```

Comment:
I agree. 
Until now, that is how it used to behave since the implementation used to have a `modal`.
I've put a comment on the PR description:
> I would like to add one more enhancement on a separate PR, that the first focusable element will be the first SkipLink element and not the container as its behaving now (and always behaved that way). I can file it on a separate PR if agreed.

Let me know what you think.

Reply:
Hmm... is that how skip links work on other sites? I think it's more important to be compatible with the general use of skip links than with prior grommet behavior in this case.

Reply:
Done on the recent commit, and I've updated the PR description.

Reply:
I'm still not sure why the Layer needs `tabIndex={0}`. I was thinking that the first `SkipLink` would be what receives the focus already.

Reply:
There are comments in SkipLinks `onFocus` that explains how the focus is being addressed, and how this tabIndex is playing an important part in it.
That being said, as part of investigating how to let `position='hidden'` do its thing and addressing the Layer issue that doesn't hide the Layer on `modal={false}`, I might be able to simplify this tabIndex behavior and remove it altogether. 
I'll keep this comment open and circle back to it once I'll have some resolutions.

Reply:
Update: 
I pushed a fix for Layer that addresses the issue of hiding the Layer when `position="hidden" && modal={false}` and that fixes the issue for SkipLinks. Thanks to this fix I was able to remove `tabIndex=0` so this concern should be out of our way now.